Jasmine/Levels/ViewControllers/GameLevelList/GameLevelListViewController.swift

```diff
@@ -30,6 +30,7 @@ class GameLevelListViewController: UIViewController {
             levelsCategoriesToggle.selectedSegmentIndex
                 = newValue ? GameLevelListViewController.builtInLevelsIndex
                            : GameLevelListViewController.customLevelsIndex
+            reloadAllData()
```

Comment:
is there a need to call this,  because it is called in `onLevelsCategoriesSwitched`?